force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -94,10 +103,13 @@ public with sharing class VariablesDataTypesOperators {
      * @return true if the number is positive, false otherwise.
      */
     public static Boolean isPositive(Integer num) {
-        return null; // Replace null with the variable you used to store the result
+        if(num > 0){
```

Comment:
You can also return this directly (num > 0) evaluates to true, so you dont need the extra if returning true/false. 
`return (num > 0);`

Reply:
Nice! Thanks for letting me know!